transports/janus_websockets.c

```diff
@@ -636,7 +636,11 @@ int janus_websockets_init(janus_transport_callbacks *callback, const char *confi
 			info.ssl_private_key_password = NULL;
 			info.gid = -1;
 			info.uid = -1;
-			info.options = 0;
+#if (LWS_LIBRARY_VERSION_MAJOR == 3 && LWS_LIBRARY_VERSION_MINOR >= 2) || (LWS_LIBRARY_VERSION_MAJOR > 3)
+                        info.options = LWS_SERVER_OPTION_FAIL_UPON_UNABLE_TO_BIND;
+#else
+                        info.options = 0;
+#endif
```

Comment:
Indentation seems broken? Please use tabs, not spaces (code style nit).
As a side note, no need for the `else` part with the `0` init: we already memset the whole `info` object to `0` when we declare the variable.